.github/workflows/demo_check.yml

```diff
@@ -0,0 +1,26 @@
+name: RxPlayer Demo Checks
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - demo/**
+
+jobs:
+  scripts_lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - run: npm install
```

Comment:
you can use `npm ci`  instead of `npm install` for safety purpose
(it preserves package.json  from edition and will fail if dependencies in package.json and package-lock mismatch)

Reply:
I just did that for all ci scripts